contribs/gmf/src/datasource/ExternalDataSourcesManager.js

```diff
@@ -336,8 +337,8 @@ export class ExternalDatSourcesManager {
   }
 
   /**
-   * @param {Object} layer WMS Capability Layer object.
-   * @param {Object} capabilities  WMS Capabilities definition
+   * @param {Object<any, any>} layer WMS Capability Layer object.
```

Comment:
Is there no better definition ? `Object{string, any}` ?

Reply:
Good catch, thanks :-)